koji/flatten-el8/koji-ansible/roles/grobisplitter/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: install python packages
+  package: name={{ item }} state=present
+  with_items:
```

Comment:
IIRC you can pass a list to `name` and have a single yum/dnf transaction which is faster.